CHANGES.md

```diff
@@ -33,11 +35,92 @@
 * [Version 1.0.1](#version-101)
 * [Version 1.0](#version-10)
 
+## Version 2.5.2 (June 15, 2020)
+
+Patch release with various bug/security fixes and build/install fixes.
+
+### Summary
+
+* Invalid input could cause a heap-use-after-free error in DeepScanLineInputFile::DeepScanLineInputFile()
+* Invalid chunkCount attributes could cause heap buffer overflow in getChunkOffsetTableSize()
+* OpenEXRConfig.h now correctly sets OPENEXR_PACKAGE_STRING to "OpenEXR" (rather than "IlmBase")
+* Various Windows build fixes
+
+### Merged Pull Requests
+
+* [755](https://github.com/AcademySoftwareFoundation/openexr/pull/755) Fix OPENXR_PACKAGE_NAME
+* [751](https://github.com/AcademySoftwareFoundation/openexr/pull/751) Disable all Windows builds temporarily.
```

Comment:
Should "release notes" cull the PRs that are only about the repo itself or the CI, removing clutter so people scanning them will only see features or behavior changes that they might experience on their end?

Reply:
Merge cases I'd argue perhaps should be culled from release notes:

* TSC meeting minutes.
* Ephemeral changes, such as PRs that make a change that is reversed by a second PR within the same release cycle.
* Routine changes to the CI system (but do list truly notable changes, like informing people that a certain build configuration or new compiler version is now thoroughly tested, which was not before).